src/less/actionable/ds4/actionable-base.less

```diff
@@ -84,3 +84,21 @@ button.img-btn {
         }
     }
 }
+
+.icon-btn--badged,
+.icon-link--badged {
+    position: relative;
+
+    .badge {
+        border: 2px solid;
+        border-color: @ds4-color-core-white;
+        font-size: @ds4-font-size-12;
```

Comment:
Since the color and font size are the only differences, is there a shared base file for actionable we could leverage? Is there a reason there isn't a shared base file for the actionable module?

Reply:
No reason, but I'd prefer to do that as a separate PR.

Reply:
Okay. Should I create an issue, or do you want to (so we don't forget about it)?

Reply:
Feel free to create one. Otherwise I'll get to it as part of my tech debt sweep.

Reply:
Created #440.